app/grandchallenge/cases/models.py

```diff
@@ -30,6 +34,9 @@
     ImageType,
     PatientSex,
 )
+from pydantic import ConfigDict, Field, field_validator
```

Comment:
Actually, I think that we'll need to add pydantic to our dependencies in `pyproject.toml` now. It's already installed but as a sub-dependency.

Reply:
I corrected the last commit. You will need to update your branch. Also, your version of `uv` is out of date so please update that.